mdbook-tera-backend/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "mdbook-tera-backend"
+version = "0.0.1"
+authors = ["Martin Geisler <mgeisler@google.com>", "Alexandre Senges <asenges@google.com>"]
+categories = ["template-engine"]
+edition = "2021"
+keywords = ["mdbook", "tera", "renderer", "template"]
+license = "Apache-2.0"
+repository = "https://github.com/google/mdbook-i18n-helpers"
+description = "Plugin to extend mdbook with Tera templates and custom HTML components."
+
+[dependencies]
+anyhow = "1.0.75"
+chrono = { version = "0.4.31", default-features = false, features = ["alloc"] }
```

Comment:
I don't think you need this dependency — please go through the others as well and remove everything not needed :slightly_smiling_face: